easybuild/easyconfigs/e/Emacs/Emacs-27.1-GCCcore-10.2.0.eb

```
@@ -0,0 +1,47 @@
+#
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# reciPY derived from preexisting reciPy for GCCcore-9.2.0.
+#
+easyblock = 'ConfigureMake'
+
+name = 'Emacs'
+version = '27.1'
+
+homepage = 'https://www.gnu.org/software/emacs/'
+description = """GNU Emacs is an extensible, customizable text editor--and more.
+ At its core is an interpreter for Emacs Lisp, a dialect of the Lisp programming
+ language with extensions to support text editing."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+checksums = ['ffbfa61dc951b92cf31ebe3efc86c5a9d4411a1222b8a4ae6716cfd0e2a584db']
+
+builddependencies = [
+    ('binutils', '2.35'),
```

```suggestion
    ('binutils', '2.35'),
    ('pkg-config', '0.29.2'),
```

I'm seeing `checking for pkg-config... /bin/pkg-config` in the log. We should use a `pkg-config` built by EasyBuild for this.